### DISCUSSION TITLE
check if email address is used for another user

### DISCUSSION THREAD

**Discussion by samaremad** [serenity-is/Serenity#5481] [Help]
HI EVERYONE 

i need to check if the email allready created for another user so that the error shown th enter another email how can i do that 

i write the below code in Usergrid but it dosnt work correct 

 this.form.Email.addValidationRule(this.uniqueName, x => {
                var e = this.form.Email.value;
                if (UsersRow.Fields.Email == e)
                    return "this email address allready created "
            }

**Comment by VictorTomaili** [comment]
Please edit title to lower case @samaremad

**Reply by samaremad** [reply]
i did it

can u please help me with issue

**Reply by VictorTomaili** [reply]
create a service endpoint for check, and call it from validation rule. Better preferred way is check it on save user and throw validation exception from server side.

**Reply by VictorTomaili** [reply]
Or just add unique to database table and userrow.
